mailpoet/lib/Automation/Engine/Data/Workflow.php

```diff
@@ -191,8 +191,8 @@ public static function fromArray(array $data): self {
       }, Json::decode($data['steps'])),
       new \WP_User((int)$data['author'])
     );
-    $workflow->id = (int)$data['id'];
-    $workflow->versionId = (int)$data['version_id'];
+    $workflow->id = (int)($data['id'] ?? 0);
+    $workflow->versionId = (int)($data['version_id'] ?? 0);
```

Comment:
A bit of a note for myself when looking at this. The change seems a bit dangerous. The `fromArray` is meant to be used when fetching data from the DB, and both ID and version ID should always be set. This seems to kind of silently convert it to `0`, which is invalid.

I guess that's related to https://github.com/mailpoet/mailpoet/pull/4376/files#diff-03a583fe00e732caecf03063cb72efe77e089b482fefee6934eaf01e9bcde84eR58, will have a look.